core/Type/Type.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Type;
+
+use Piwik\Cache;
+use Piwik\CacheId;
+use Piwik\Plugin;
+use Piwik\Plugin\Manager as PluginManager;
+use Piwik\Property\PropertySettings;
+
+class Type
+{
+    const ID = '';
+    protected $name = 'General_Property';
+    protected $namePlural = 'General_Properties';
+    protected $description = 'Default type';
+    protected $howToSetupUrl = '';
```

Comment:
See comment here: https://github.com/piwik/piwik/issues/7893#issuecomment-109113915 I think it makes a lot of sense to fall back to a universal type if it is removed and we will still need some kind of wording for it, even if it is ignored etc.